src/hera.cpp

```diff
@@ -132,6 +137,38 @@ pair<evmc_status_code, bytes> callSystemContract(
   return {result.status_code, ret};
 }
 
+pair<evmc_status_code, bytes> locallyExecuteSystemContract(
+  evmc_context* context,
+  evmc_address const& address,
+  int64_t & gas,
+  bytes_view input,
+  bytes_view code,
+  bytes_view state_code
+) {
+  const evmc_message message = {
+    .kind = EVMC_CALL,
+    .flags = EVMC_STATIC,
+    .depth = 0,
+    .gas = gas,
+    .destination = address,
+    .sender = {},
+    .input_data = input.data(),
+    .input_size = input.size(),
+    .value = {},
+    .create2_salt = {},
+  };
+
+  unique_ptr<WasmEngine> engine = wasmEngineCreateFn();
+  ExecutionResult result = engine->execute(context, code, state_code, message, false);
+
+  bytes ret;
+  evmc_status_code status = result.isRevert ? EVMC_REVERT : EVMC_SUCCESS;
+  if (status == EVMC_SUCCESS && result.returnValue.size() > 0)
```

Comment:
```suggestion
  if (status == EVMC_SUCCESS && !result.returnValue.empty())
```
but the `returnValue` is not really needed.